publisher/publisher.go

```diff
@@ -316,6 +316,15 @@ func (pub *Impl) singleLogSubmit(
 	if err != nil {
 		return nil, err
 	}
+	timestamp := time.Unix(int64(sct.Timestamp)/1000, 0)
+	if timestamp.Sub(time.Now()) > time.Minute {
```

Comment:
I think the `publisher` changes that return an error for an SCT too far in the future or the past need a corresponding unit test.